tests/integrations/wsgi/test_wsgi.py

```diff
@@ -67,6 +67,29 @@ def test_basic(sentry_init, crashing_app, capture_events):
     }
 
 
+def test_basic_script_name_is_respected(sentry_init, crashing_app, capture_events):
+    sentry_init(send_default_pii=True)
+    app = SentryWsgiMiddleware(crashing_app)
+    client = Client(app)
+    events = capture_events()
+
+    with pytest.raises(ZeroDivisionError):
+        # setting url with PATH_INFO: bark/, HTTP_HOST: dogs.are.great and SCRIPT_NAME: woof/woof/
+        client.get("bark/", "https://dogs.are.great/woof/woof/")
+
+    (event,) = events
```

Comment:
```suggestion
    assert len(events) == 1
    (event,) = events
```
Assuming we are expecting to have exactly one event, we should explicitly assert this expectation to document this expectation in case the test fails. Otherwise, if there isn't exactly one event, the `(event,) = event` line will error, and it might be unclear to whoever is debugging the error whether there is an error in the code or whether the test is implemented incorrectly.